ivadomed/config/config_sctTesting.json

```diff
@@ -90,7 +90,7 @@
         "ROICrop2D": {
             "size": [48, 48]
         },
-        "ToTensor": {},
+        "NumpyToTensor": {},
         "NormalizeInstance": {},
```

Comment:
Should probably add applie_to: "img" as well

Reply:
What do you mean? I thought you suggested that when "applied_to" is not mentioned then the transform is applied to all kind of images. Have I missed something? :)
Also: this config file is not updated yet (cf comment TODO "Update config files" --> only config_small is ready, cf Slack message). Sorry for wasting your time, I should not have asked you to review it before all ready.

Reply:
Ah yes you are right I see it in your TODO list now :) I just meant to add "applied_to":"img"  as parameter to "NormalizeInstance" in all config files since we never want to normalize the binarized mask. But totally my bad, I missed this element on your todo :). Not a waste of time at all! As said here:

> Wouhou awesome refactoring :)! Definitely worth the time. I was able to run everything, but since the orientation were not correct for sagittal couldn't fully test CenterCrop and Resample. Runs well in multichannel and multilabel!

Everything works very well! Once it's merged with master I'll test those transforms :rocket:

Reply:
done :)